### DISCUSSION TITLE
Download audiobook to device

### DISCUSSION THREAD

**Discussion by PinkMusen** [advplyr/audiobookshelf-app#783] [Ideas]
Sometimes you may be away from wifi or have limited data, it would be nice to be able to download an audiobook (or more) to the device so that it can play from local storage instead of streaming it like the audible app does.
Is this something that's planned for the future ?

**Comment by advplyr** [comment]
This is already supported for both android and iOS

**Comment by hyyyjinx** [comment]
@advplyr I think this might be a regression in the latest iOS app? I filed a ticket with details here: 

https://github.com/advplyr/audiobookshelf-app/issues/788
